test/test_cpp_extensions_stream_and_event.py

```diff
@@ -0,0 +1,108 @@
+# Owner(s): ["module: unknown"]
```

Comment:
We should not add new unowned tests to CI

Reply:
I created a "module: mtia" (https://github.com/pytorch/pytorch/labels/module%3A%20mtia) that you should use there.
Let's discuss offline how to make sure someone is looking at it!